source/installation/install-on-a-single-machine/install-on-a-single-machine.txt

```diff
@@ -41,6 +41,9 @@ If you are migrating from an Oracle database:
 
 - Download `21.6.0.0 of odjbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`_. 
 
+- (Optional) If you are migrating tables that contain the ``XML`` type, you must also download the 
+  `xdb.jar <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`_ driver. 
```

Comment:
[nit]

I think our convention is to put two underlines after the closing bracket for external links:

```suggestion
- (Optional) If you are migrating tables that contain the ``XML`` type, you must also download the 
  `xdb.jar <https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html>`__ driver. 
```

Reply:
For context, not putting two underlines used to throw a build error, but I think the parser has been updated to accept either one or two underlines. This suggestion is mostly to standardize the link formatting with most of our other links.

Reply:
Done.